PhysicsTools/PatAlgos/python/slimming/slimming_cff.py

```diff
@@ -62,6 +62,12 @@
 from Configuration.Eras.Modifier_pp_on_AA_2018_cff import pp_on_AA_2018
 pp_on_AA_2018.toReplaceWith(slimmingTask, slimmingTask.copyAndExclude([slimmedOOTPhotons]))
 
+from Configuration.Eras.Modifier_pp_on_PbPb_run3_cff import pp_on_PbPb_run3
+from PhysicsTools.PatAlgos.packedPFCandidateMuonID_cfi import packedPFCandidateMuonID
+(pp_on_AA_2018 | pp_on_PbPb_run3).toModify(packedPFCandidateMuonID, muonSelectors = cms.vstring(["AllTrackerMuons", "TMOneStationTight"]))
```

Comment:
this is not particularly HI-specific, it should be OK to just have these two selectors in the fillDescriptions directly.

If we still keep it with an era, please drop the type specs:
```suggestion
(pp_on_AA_2018 | pp_on_PbPb_run3).toModify(packedPFCandidateMuonID, muonSelectors = ["AllTrackerMuons", "TMOneStationTight"])
```

Reply:
Moved the two selectors to the fillDescriptions.